src/include/common/constants.h

```diff
@@ -11,7 +11,9 @@ namespace terrier {
  * in code.
  */
 struct Constants {
-  // Should only ever be a power of 2
+  /**
+   * Should only ever be a power of 2.
```

Comment:
This shouldn't be a doxygen comment as that is an implementation detail. Back to inline?